### ISSUE TITLE
Full-width Comma in Resume English Template 

### ISSUE THREAD

**Issue by jmnie** [billryan/resume#41]
Hi, in the English template, the default comma is Chinese full-width comma, which should be half-width commas.

Excellent Work

**Comment by billryan** [comment]
Could you point it out? I can not find it.

**Comment by jmnie** [comment]
In the figure following, I used $,$ 
![comma_1](https://user-images.githubusercontent.com/11575060/64479362-17ff3400-d184-11e9-96da-be71431ff52d.png)
On the following is the default comma in English template:
![comma_2](https://user-images.githubusercontent.com/11575060/64479366-3a914d00-d184-11e9-86f4-0d555f9bf4ac.png)

Actually it's not a bug, from the example images in your repository I found the comma was half-width. Some recruiters viewed my resume said it should be the case I.
I used MikTeX compiled on ubuntu 18.04. It should be a problem related to my environment. 
Thanks
